x-pack/legacy/plugins/siem/public/components/charts/common.tsx

```diff
@@ -138,10 +139,10 @@ export const getTheme = () => {
       barsPadding: 0.5,
     },
   };
-  const isDarkMode = chrome.getUiSettingsClient().get('theme:darkMode');
+  const isDarkMode = chrome.getUiSettingsClient().get(DEFAULT_DARK_MODE);
   const defaultTheme = isDarkMode ? DARK_THEME : LIGHT_THEME;
   return mergeWithDefaultTheme(theme, defaultTheme);
 };
 
-const kibanaTimezone = chrome.getUiSettingsClient().get('dateFormat:tz');
+const kibanaTimezone = chrome.getUiSettingsClient().get(DEFAULT_DATE_FORMAT_TZ);
```

Comment:
Same here with the `any` as well.

Reply:
Yep, let's get rid of those to not pollute the project space.